src/main/scala/nl.knaw.dans.easy.dd2d/DepositIngestTask.scala

```diff
@@ -70,10 +78,23 @@ case class DepositIngestTask(deposit: Deposit, dansBagValidator: DansBagValidato
              debug("Keeping dataset on DRAFT")
              Success(())
            }
+      _ <- instance.dataset(persistentId).awaitUnlock()
```

Comment:
You have now made it possible to give for depositing its own "locking parameters".  Why not use them?
`instance.dataset(persistentId).awaitUnlock(publishAwaitUnlockMaxNumberOfRetries, publishAwaitUnlockMillisecondsBetweenRetries)`

Reply:
Good catch :)